mm-redmine.psm1

```diff
@@ -281,7 +295,9 @@ function Get-RedmineIssue {
     $sortOrder = if ($SortDesc) { ":desc" }
     $sort      = if ($SortColumn) { "&sort={0}{1}" -f $SortColumn, $sortOrder }
     $pInclude  = if ($Include) { "&include={0}" -f ($Include -join ',') }
-    if ($Id) { $pInclude = $pInclude.Replace('&', '?') }
+    if ($Id) { 
+        if ($include) {$pInclude = $pInclude.Replace('&', '?') }
```

Comment:
Not a good form - if within if can be avoided. `$Include` is also a function param, by convention in this repo all function params use large first letter.

What was wrong with previous one anyway ? If `$Include` was not present it would replace nothing and we would have empty variable.

Reply:
In my use case it caused me some problems in error processing, cause it throws `You cannot call a method on a null-valued expression.`  But I can revert the changes if they are not more widely applicable.

Reply:
The actual code I should have written is `if ($Id -and $Include) { $pInclude = $pInclude.Replace('&', '?') }`

Reply:
Replace requires string object indeed.

Reply:
Yes.. use alternative code.